swingset/src/main/java/com/nqadmin/swingset/SSDBComboBox.java

```diff
@@ -1632,6 +1632,10 @@ public void updateSSComponent() {
 			if (!hasItems()) {
 				return;
 			}
+      
```

Comment:
This method had already been scrubbed/shortened, but I guess you didn't have that code. setSelectionPending(true) seemed like the only new thing and I placed that before the call to getBoundColumnText();

Reply:
`setSelectionPending(true)` instead of `getEditor().setItem("")`

Oh, you had already gotten rid of `setItem("")`. Yes, I put it before `getBoundColumnText();`

Reply:
"This method" is updateSSComponent() and you have to click the blue up/down arrow to expand it in review mode.